### ISSUE TITLE
Can't Place in creative mode?

### ISSUE THREAD

**Issue by Winddbourne** [HenryLoenwind/extrabedrock#2]
I installed the mod thinking I would use it to create a pretty, unbreakable, spawn location for my skyblock map. This would have let me have no spawn protection in place so people could place vending blocks for stores and otherwise meet and mingle at "spawn". 

But when I went into the game to use structure blocks and your bedrock variants I found that there was no tab in JEI, and the blocks couldn't even be found searching the creative inventory. I had to uninstall simply because I could see no way of using these blocks in the actual game. lol

**Comment by HenryLoenwind** [comment]
Did you even try to read the curseforge page?

**Comment by Winddbourne** [comment]
Yes, I read the page. It's nice that there are extra ways to use the
blocks; that should never mean that the default methods provided by
Minecraft for accessing blocks aren't included. Nobody has to say that
blocks added by a mod can be accessed and used from the creative menu; if a
block can't be used that way you specify that in the description; just as
you mention extra ways that they can be used.

Also, none of the methods you list will be useful to the target audience
you stated. A mod pack maker needs to be able to make and spawn structures
using your blocks, a map maker may need to place them around a set world
border or use them in strategic locations to make the map's more
challenging. Most world gen tools don't take block states, they use the
structure blocks included by minecraft, or take block ID's for random
placement in a terrain generator . . .  Recurrent complex uses it's own
structure block alternative, biometweaker places things randomly, I'm
trying to use Void Island Control right now; JED works like biometweaker
(using that one too).


The setblock command MIGHT do what I want but building any sort of large
structure by repeatedly typing a command would be super slow and
frustrating . . . I'm pretty sure I'm completely misunderstanding how it
works unless it has more arguments and conditional setting that aren't
included on the page; which is why I checked the WIKI before listing a lack
of normal creative mode support as a problem. I was hoping that there were
arguments to the command for setting multiple blocks in an area, shape
settings for things like spheres, cubes, and cylinders, etc . . . that
still would be clunky and weird but I might be able to use it without
killing myself . . . I've got a lot of JSON coding and playtesting I want
to do. Nobody has time to place blocks repeatedly using a command. As I
said I'm pretty sure I'm completely misunderstanding what that thing is
supposed to do. I'll blame it on my autism and just ask for an accessible
interface. :)
